packages/svelte/src/internal/client/dom/template.js

```diff
@@ -187,26 +187,41 @@ function run_scripts(node) {
 	// scripts were SSR'd, in which case they will run
 	if (hydrating) return;
 
+	const is_fragment = node.nodeType === 11;
 	const scripts =
 		/** @type {HTMLElement} */ (node).tagName === 'SCRIPT'
 			? [/** @type {HTMLScriptElement} */ (node)]
 			: node.querySelectorAll('script');
+	const effect = /** @type {Effect} */ (current_effect);
+
 	for (const script of scripts) {
-		var clone = document.createElement('script');
+		const clone = document.createElement('script');
```

Comment:
`const`?

Reply:
yes, because they are used in a closure which may be delayed, and with `var` all invocations would get the latest value in the array